### DISCUSSION TITLE
Annotation services for pose?

### DISCUSSION THREAD

**Discussion by timsainb** [open-mmlab/mmpose#2770] [Q&A]
Hello, I am trying to find a good annotation service to label key points on animals. We have used scale.ai but had issues getting data >20 keypoints labeled. Does anyone have recommendations for labeling services that work well for pose data?

**Comment by smidm** [comment]
I use https://app.cvat.ai/. The UX and documentation could be better, but it's doing the job well. It supports skeletons, that you can use to lay all keypoints at once to approximate positions. It's open source, you can always run it on your own.
